src/compas_model/notebook/scene/modelobject.py

```diff
@@ -0,0 +1,17 @@
+import compas_model.model
+from compas_notebook.scene import ThreeSceneObject
+
+
+class ThreeModelObject(ThreeSceneObject):
+
+    def __init__(self, model: compas_model.model.Model, **kwargs):
+        super().__init__(item=model, **kwargs)
+        self.model = model
+        self.show_hierarchy = False
```

Comment:
What would be the most basic interactions visualization applicable to all elements?

Cases:
a) when a pair of elements are visualized, they are colored 
b) but how you would display adjacencies between multiple or all elements (without using the form)?